modules/developers-guide/pages/glossary.adoc

```diff
@@ -8,53 +8,45 @@ actor::
-  The first time you use the {sdk-short-name}, the `+dfx+` command-line tool creates a `+default+` developer identity for you with a public/private key pair in a PEM file. This {IC} developer identity is represented internally by a derived **principal** data type and a textual representation of the principal often referred to as your **principal identifier**.
-  The developer identity can also be used to derive an **account identifier**—similar to a Bitcoin or Ethereum address—to hold ICP tokens on your behalf in the {IC} ledger canister.
+  A physical or virtual machine that hosts all the hardware, replica software, and configuration settings required to participate in the {IC} blockchain.
 
 [[g-replica]]
 replica:: 
-  In the context of the {platform}, a replica refers to the {IC} processes (for example, the `+replica+`, `+nodemanager+`, and other lower-level {IC} protocol processes) running on a physical computer node in the network.
+  In the context of the {IC} blockchain, a replica refers to the {IC} protocol processes (for example, the `+replica+`, `+nodemanager+`) running on a node.
   For the {sdk-short-name}, you use the `+dfx start+` and `+dfx stop+` commands to start and stop the `+replica+` process locally to provide a local network for development.
 
-[[g-smart-contract]]
-smart contract:: 
-  A smart contract is software that enables trusted transactions and agreements to be carried out across a distributed, decentralized blockchain network without the need for any central authority or legal system.
-  With a smart contract, the terms of a transaction or agreement are written directly into lines of code that are executed on the blockchain network. 
-  The code controls the execution, and the transactions are tamper-proof, traceable, and irreversible.
-  On the {IC}, smart contracts are deployed as <<g-canister,canisters>>.
-
-[[g-wallet]]
-wallet::
-  On the {IC}, a wallet is a specialized application that allows you to store and retrieve your digital assets.
-  The wallet application is implemented as a canister that runs on the {IC}. A wallet enables you to manage your ICP token balances, convert ICP token into cycles, and distribute cycles to your own or other users' canisters as a means for accessing or providing internet services.
-
 [[g-wasm]]
 WebAssembly::
   https://webassembly.org/[WebAssembly] (`+Wasm+`) is a low-level computer instruction format. 
-  Because WebAssembly defines a portable,open-standard, binary format that abstracts cleanly over most modern computer hardware, it is broadly supported for programs that run on the internet. 
-  Programs written in {proglang} are compiled to WebAssembly code for execution on {IC} replicas.
+  Because WebAssembly defines a portable, open-standard, binary format that abstracts cleanly over most modern computer hardware, it is broadly supported for programs that run on the internet.
+  Programs written in {proglang} are compiled to WebAssembly code for execution on replicas.
```

Comment:
```suggestion
  Dapps written in {proglang} are compiled to WebAssembly code for execution on the {IC} blockchain.
```
Maybe like this?

Reply:
Github won't let me apply this suggestion. Will manually apply.